src/components/Misc/InAppThemeProvider.tsx

```diff
@@ -1,20 +1,32 @@
-import React, { useState, useEffect } from 'react';
-import Cookies from 'js-cookie';
+import React from 'react';
+import Cookies from 'universal-cookie';
+import { IncomingHttpHeaders } from 'http';
 import { ThemeProvider } from 'emotion-theming';
+import { Global, css } from '@emotion/core';
+import {
+  defaultTheme, darkTheme, resetStyles, RIDITheme,
+} from 'src/styles';
 
-import { defaultTheme, darkTheme } from 'src/styles';
+const inappStyle = (theme: RIDITheme) => css`
+  html {
+    background-color: ${theme.backgroundColor};
+  }
+  ${resetStyles};
+`;
 
-const InAppThemeProvider = ({ children }) => {
-  const [theme, setTheme] = useState('');
-  useEffect(() => {
-    setTheme(Cookies.get('ridi_app_theme') || '');
-  });
-
-  return (
-    <ThemeProvider theme={theme === 'dark' ? darkTheme : defaultTheme}>
-      {children}
-    </ThemeProvider>
-  );
+export type Theme = 'dark' | '';
+export const getAppTheme = ({ cookie }: IncomingHttpHeaders): Theme => {
+  const cookies = new Cookies(cookie);
+  const theme = cookies.get('ridi_app_theme');
+  if (theme === 'dark') {
+    return theme;
+  }
+  return '';
 };
-
+const InAppThemeProvider: React.FC<{ theme: Theme }> = ({ children, theme }) => (
+  <ThemeProvider theme={theme === 'dark' ? darkTheme : defaultTheme}>
+    <Global styles={inappStyle} />
```

Comment:
```suggestion
    <Global styles={[resetStyles, inappStyle]} />
```

이렇게 하고 `inappStyle`에서 `resetStyles`를 빼는 게 어떨까요?